small/matras.c

```diff
@@ -335,6 +335,33 @@ matras_dealloc_range(struct matras *m, matras_id_t range_count)
 	matras_dealloc(m);
 }
 
+/**
+ * Reserve the max amount of extents required to successfully touch @p count
+ * blocks. The extents are reserved in the allocator given on construction.
+ */
+int
+matras_reserve_touches(struct matras *m, unsigned count)
```

Comment:
For some reason, the function name sounds awkward to me. Let's maybe rename it to `matras_touch_reserve`?

Reply:
Renamed to `matras_touch_reserve`.